setup.py

```diff
@@ -30,7 +30,7 @@
     packages=find_packages(exclude=['tests*']),
     package_data={'awscli': ['data/*.json', 'examples/*/*.rst',
                              'examples/*/*/*.rst', 'topics/*.rst',
-                             'topics/*.json']},
+                             'topics/*.json', 'bin/aws_bash_completer']},
```

Comment:
@jamesls I'm not sure whether `bin/aws_bash_completer` belongs to `package_data` or to `scripts`. It's not executable file so I decided to keep it to `package_data` for now.